openml/extensions/sklearn/extension.py

```diff
@@ -843,7 +865,8 @@ def _deserialize_model(
                     del parameter_dict[param]
         return model_class(**parameter_dict)
 
-    def _check_dependencies(self, dependencies: str) -> None:
+    def _check_dependencies(self, dependencies: str,
+                            strict_version: Optional[bool] = True) -> None:
```

Comment:
Is there a reason why `strict_version` is a keyword argument? By making it a positional argument we could make sure that one cannot forget to specify the behavior in the future?

Reply:
You want to make it a positional-only argument? Isn't that like a python3.8 feature?

Reply:
[Yes](https://docs.python.org/3.8/whatsnew/3.8.html#positional-only-parameters). I'm guessing he meant not to make it an optional argument, but a mandatory one.

Reply:
Thanks @PGijsbers that's corerct, I meant making it a mandatory argument.